backend/nbp_edo.test.js

```diff
@@ -0,0 +1,99 @@
+const EDO = require("./nbp_edo")
+const dayjs = require("dayjs")
+const Dinero = require("dinero.js")
+
+// sampleEDO returns EDO bond with:
+// value = 100 PLN
+// first_year_roi = 2.7%
+// margin = 1.5% (marza)
+// inflation = 2.5% 
+let pln = (amount) => Dinero({ amount: amount, currency: 'PLN' });
+let sampleEDO = (start) => EDO.createBond(pln(10000), 2.7, 1.5, 2.5, start);
+let investment_start = () => dayjs("2018-04-04T16:00:00.000Z").startOf('day');
+
+const expected = [
+    JSON.stringify(pln(10270)),
+    JSON.stringify(pln(10681)),
+    JSON.stringify(pln(11108)),
+    JSON.stringify(pln(11552)),
+    JSON.stringify(pln(12014)),
+    JSON.stringify(pln(12495)),
+    JSON.stringify(pln(12995)),
+    JSON.stringify(pln(13515)),
+    // TODO: probably one cent extra is returned to be checked, but we dont care for now
+    JSON.stringify(pln(14056)),
+    JSON.stringify(pln(14618)),
+];
+
+test('at the very start the history is empty', () => {
+    // given
+    const bond = sampleEDO(investment_start());
+    // when
+    const history = EDO.edoHistory(bond, investment_start());
+    // then
+    expect(history.length).toBe(0);
```

Comment:
We should consider returning 100pln here, because this is the initial value of the bond. In case of creating a graph we will need this value anyway.